src/components/TopologySidebar/ComponentChecks.tsx

```diff
@@ -0,0 +1,112 @@
+import { useQuery } from "@tanstack/react-query";
+import { useAtom } from "jotai";
+import { useEffect } from "react";
+import { AiFillHeart } from "react-icons/ai";
+import { Link } from "react-router-dom";
+import {
+  ComponentHealthCheckView,
+  getComponentChecks
+} from "../../api/services/topology";
+import PillBadge from "../Badge/PillBadge";
+import CollapsiblePanel from "../CollapsiblePanel";
+import EmptyState from "../EmptyState";
+import TextSkeletonLoader from "../SkeletonLoader/TextSkeletonLoader";
+import { refreshButtonClickedTrigger } from "../SlidingSideBar";
+import { Status } from "../Status";
+import Title from "../Title/title";
+import { Icon } from "../Icon";
+
+type ComponentCheckLinkProps = {
+  componentCheck: ComponentHealthCheckView;
+};
+
+export function ComponentCheckLink({
+  componentCheck: check
+}: ComponentCheckLinkProps) {
+  return (
+    <Link
+      to={{
+        pathname: `/health`,
+        search: `?checkId=${check.id}&timeRange=1h`
+      }}
+      className={`flex flex-row justify-between w-full items-center space-x-2 p-2 rounded-md hover:bg-gray-100`}
+    >
+      <div className="flex flex-row gap-2 w-full items-center">
+        <div className="flex flex-row space-x-1 items-center flex-1text-sm ">
+          <Status good={check.status === "healthy"} />
```

Comment:
Shouldn't we be passing through the check here ?

Reply:
The component has two props - good and mixed, I am guessing mixed is for grouped table columns

Reply:
At the moment it may be just the 2, but by passing the check through we have the option in the future of adding more (silenced, flapping, stale, etc) without having to change everywhere its used.